packages/core/src/kernelCoreTemplate.js

```diff
@@ -74,24 +74,19 @@
     const scuttleGlobalThisDefaults = {
       enabled: true,
       exceptions: [],
-      recursive: false,
+      scuttler: '',
```

Comment:
weird default

Reply:
this prop is expected to be a string, this is the default value of type:string (thought it would make sense to stick to one type if possible)